core/src/test/resources/test_assets.json

```diff
@@ -53,10 +61,16 @@
         "fields": {
           "transaction": {
```

Comment:
Can we remove all uses of `fields.transaction`? We don't support this in the platform so it is confusing to require this object.

Reply:
Let's address this in a separate PR as well. I created ANCHOR-383